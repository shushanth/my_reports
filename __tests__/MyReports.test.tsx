import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import MyReports from "@/app/my-reports/page";

describe("MyReports", () => {
  it("renders component", async () => {
    render(<MyReports />);
    const heading = screen.queryByTestId("myReports_page--component");
    expect(heading).toBeDefined();
  });

  it("renders component page unchanged", () => {
    const { container } = render(<MyReports />);
    expect(container).toMatchSnapshot();
  });
});
