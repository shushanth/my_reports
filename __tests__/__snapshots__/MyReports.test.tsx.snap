// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyReports renders component page unchanged 1`] = `<div />`;
